docs/reference/api/platform/Logger.md

```diff
@@ -154,9 +154,6 @@ Usage:
 #define tr_cmdline(...)     MBED_LOG(TRACE_LEVEL_CMD, TRACE_GROUP, ##__VA_ARGS__)
 ```
 
-### Log Helper functions
```

Comment:
Are we removing these helper functions?

Reply:
We are not removing helper functions, but helper functions are not API's. They will stay as internal functions and are subject to change.